zebra/zebra_rnh.h

```diff
@@ -59,6 +59,9 @@ extern void zebra_evaluate_rnh(struct zebra_vrf *zvrf, afi_t afi, int force,
 extern void zebra_print_rnh_table(vrf_id_t vrfid, afi_t afi, struct vty *vty,
 				  enum rnh_type type, struct prefix *p);
 extern char *rnh_str(struct rnh *rnh, char *buf, int size);
+extern int send_client(struct rnh *rnh, struct zserv *client,
```

Comment:
I'm not really keen on exposing this in this way (and definitely not named "send_client()"). this is a pretty internal api, and may not be appropriate for general use - and I'm not sure the rnh_lookup api is either, really. can you say a bit about why this is the right/only way to accomplish the thing that the sr module is trying to do?

Reply:
Also good point! This is related to update notification mechanisms for nexthops which we might revise as you suggested in the other comment.